clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -274,10 +234,9 @@ func (p *podStartupLatencyMeasurement) checkPod(_, obj interface{}) {
 		p.lock.Lock()
 		defer p.lock.Unlock()
 		key := createMetaNamespaceKey(pod.Namespace, pod.Name)
-		if _, found := p.watchTimes[key]; !found {
-			p.watchTimes[key] = metav1.Now()
-			p.createTimes[key] = pod.CreationTimestamp
-			p.nodeNames[key] = pod.Spec.NodeName
+		if _, found := p.podStartupEntries.Get(key, createPhase); !found {
```

Comment:
watchPhase

Reply:
We are setting createPhase and watchPhase. IMHO it shouldn't be any difference... Or, I'm missing something?

Reply:
good point - there shouldn't be any difference (I added this comment because it changed)